doc/version_history.rst

```diff
@@ -25,6 +25,7 @@ Requires:
 v1.11.3
 =======
 * Refactor the other test cases to contain less duplicate code.
+* Rename the 'searchZeroShutter' command to 'home' and make it apply to all subsystems.
```

Comment:
uhm.. not sure this change will classify as a patch.. consider making this v1.12.0

Reply:
Done